recipes/crc64iso/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2018 The Python Packaging Authority
```

Comment:
This seems really wrong considering the upstream repo isn't under PyPA, but it is what upstream says...

Reply:
Yeah, I am not sure if the upstream will fix this given its inactivity, but this package is quite handy for computing the crc checksum for protein sequences. Thanks for your review!